cmd/opcua/main.go

```diff
@@ -222,15 +224,13 @@ func subscribeToOpcuaServers(svc opcua.Service, nodes string, cfg opcua.Config,
 			return
 		}
 
-		if len(l) < 4 {
+		if len(l) < 2 {
```

Comment:
What is this magic number? Consider moving it in the const documented with a comment.